src/topology/opens.lean

```diff
@@ -109,6 +113,22 @@ end
 lemma supr_def {ι} (s : ι → opens α) : (⨆ i, s i) = ⟨⋃ i, s i, is_open_Union $ λ i, (s i).2⟩ :=
 by { ext, simp only [supr, opens.Sup_s, sUnion_image, bUnion_range], refl }
 
+@[simp] lemma supr_s {ι} (s : ι → opens α) : ((⨆ i, s i : opens α) : set α) = ⋃ i, s i :=
+by simp [supr_def]
+
+theorem mem_Union {ι} {x : α} {s : ι → opens α} : x ∈ supr s ↔ ∃ i, x ∈ s i :=
```

Comment:
I don't know the naming conventions here off the top of my head, but is this name right? At least there are no nearby declarations that convince me it is.

Reply:
I've changed this to `mem_supr`.